src/utils/issue.ts

```diff
@@ -198,7 +195,9 @@ export async function getAvailableOpenedPullRequests(context: Context, username:
 
   for (let i = 0; i < openedPullRequests.length; i++) {
     const openedPullRequest = openedPullRequests[i];
-    const reviews = await getAllPullRequestReviews(context, openedPullRequest.number);
+    const owner = openedPullRequest.html_url.split("/")[3];
+    const repo = openedPullRequest.html_url.split("/")[4];
```

Comment:
consider creating a helper function that parses the url and also checks for array length